### DISCUSSION TITLE
What should the default setting for ShowFullScreen be?

### DISCUSSION THREAD

**Discussion by Pentaphon** [sylikc/jpegview#125] [Polls]
It is currently "true". 

But I personally think a lot of people tend to change this. 

What would you make it?

**Comment by Pentaphon** [comment]
General issue has been opened here: https://github.com/sylikc/jpegview/issues/140

Keep voting!

**Comment by pureby** [comment]
I really dislike it when any "casual" app draws over my taskbar without permission. IMHO, entering full-screen mode should always require a user-initiated action (unless it's a game).
